internal/cli/clusters/create.go

```diff
@@ -142,6 +153,27 @@ func (opts *CreateOpts) newFlexCluster() (*atlasv2.FlexClusterDescriptionCreate2
 	return cluster, nil
 }
 
+func (opts *CreateOpts) RunDedicatedClusterLatest() error {
```

Comment:
same about latest

Reply:
thanks! I'll add in my follow-up PR since this is minor